src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -517,14 +517,14 @@ protected override void ExecuteCore(DynamoModel dynamoModel)
             protected override void SerializeCore(XmlElement element)
             {
                 var helper = new XmlElementHelper(element);
-                helper.SetAttribute("XmlFilePath", XmlFilePath);
+                helper.SetAttribute("FilePath", FilePath);
```

Comment:
The changes to the (Xml)FilePath attribute were made because there was a request to remove the "Xml" part due to the files not necessarily being xml any more. If the attribute is not changed everywhere it is possible that there could need to be switching code somewhere else, which could also be potentially problematic. With some higher level testing it appears to not be an issue, but further testing will most probably be required.